src/thor/dijkstras.cc

```diff
@@ -498,7 +501,7 @@ void Dijkstras::ExpandForwardMultiModal(GraphReader& graphreader,
     uint8_t restriction_idx = -1;
     if (directededge->IsTransitLine()) {
       // Check if transit costing allows this edge
-      if (!tc->Allowed(directededge, pred, tile, edgeid, 0, 0, restriction_idx)) {
+      if (!tc->Allowed(directededge, false, pred, tile, edgeid, 0, 0, restriction_idx)) {
```

Comment:
nit: here `false` value is used directly, in the `ExpandInner` function above `is_dest` is declared. Does it make sense to implement it consistently?

Reply:
false -> is_dest